packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -316,13 +346,42 @@ export default function NewCommentModal({
           aria-label="New comment box"
         />
       </div>
+      {getShouldLoadTooltip() && (
+        <BaseTooltip
+          content={
+            <RecommendedMention
+              users={mentionQuery !== undefined && mentions}
+              selected={selected}
+              onClick={onMentionClick}
+            />
+          }
+          offset={[getOffsetPerChar(offsetX), getOffsetPerLine(offsetY) + -90]}
```

Comment:
Does this not contradict when people are zoomed/using weird fonts? 
(not sure, might be worth checking)

Reply:
Looks like we're good to go! 

![Screen Shot 2022-03-07 at 10 37 46 PM](https://user-images.githubusercontent.com/13744167/157054913-e1b3cbe1-f8c1-42b4-b945-d4cc16ceba15.png)